client/src/components/Navbar.jsx

```diff
@@ -89,19 +89,19 @@ const NavBar = () => {
               }}
               PaperProps={{
                 sx: {
-                  backgroundColor: "brown.100",
+                  backgroundColor: "background.dark",
                   width: 300,
                   p: 1,
                 },
               }}
             >
               {pages.map((page, i) => (
-                <MenuItem key={i} onClick={handleCloseNavMenu} bgcolor="black">
+                <MenuItem key={i} onClick={handleCloseNavMenu}>
```

Comment:
best to use a key that's not index, whether thats a title, url, page object, etc